README.md

```diff
@@ -1,4 +1,30 @@
 # WORLD EMOJI DAY
+# Table Of Contents
+* [Introduction](#Introduction)
+* [UX](#UX)
+    * [User Goals](#User-Goals)
+    * [User Expectations](#User-Expectations)
+    * [Wireframes](#Wireframes)
+* [Features](#Features)
+    * [Navigation Bar](#Navigation-Bar)
+    * [Website Theme Logo](#Website-Theme-Logo)
+    * [Gallery](#Gallery)
```

Comment:
I dont think we have a Gallery feature, I think this should be Guide.

Reply:
You're right, it should be - @MihaiElisei I sent you that other README so you could look at the syntax, not copy and paste it exactly